### ISSUE TITLE
ActorGroup example results in only one actor being used.

### ISSUE THREAD

**Issue by bsutton** [renatoathaydes/actors#14]
I think the examples use of ActorGroup is problematic:

```
// create a group of 4 actors
final group = ActorGroup(Two(), size: 4);
print(await group.send(5)); // 10
group.close();
```
and
```
// or an ActorGroup
messenger = ActorGroup(Two(), size: 2);
print(await messenger.send(4)); // 8
print(await messenger.send(5)); // 10
messenger.close();
```

The above methods will result in only a single actor being used within the group.
If I've understood correctly calling `await message.send(4)` waits for the actor to complete.
The result is that send(4) and send(5) are serialised with only a single actor in the group being used at any point in time. This also reflects my experience as I've just being debugging a problem where only a single actor was being run.

I believe the correct method is to call using unawaited or storing the results in a list of futures
if you need to process the responses when the complete.
As the examples stand there is no point using ActorGroup with a roundrobin strategy over a single Actor.

```
// or an ActorGroup
messenger = ActorGroup(Two(), size: 2);
unawaited(messenger.send(4)));
var saveFuture =  messenger.send(5)); // 10
messenger.close();
```

**Comment by bsutton** [comment]
I've just tried the above changes and it doesn't seem to work.

I removed the await on the send and placed an await on the group.close.

The result is that the main thread hung on the group.close() (as expected) however none of the actors start.
If I don't await the group.close() I get a stream exception telling me that the stream has closed. 
I assume that calling group.close() closes the stream of messages that I've sent so when an actor wakes up to read the next one in the list the stream has been closed under it.
